cli/configssh.go

```diff
@@ -505,9 +527,16 @@ func (r *RootCmd) configSSH() *clibase.Cmd {
 		},
 		{
 			Flag:        "ssh-host-prefix",
-			Env:         "",
+			Env:         "CODER_CONFIGSSH_SSH_HOST_PREFIX",
 			Description: "Override the default host prefix.",
-			Value:       clibase.StringOf(&userHostPrefix),
+			Value:       clibase.StringOf(&sshConfigOpts.userHostPrefix),
+		},
+		{
+			Flag:        "wait",
+			Env:         "CODER_CONFIGSSH_WAIT", // Not to be mixed with CODER_SSH_WAIT.
+			Description: "Specifies whether or not to wait for the startup script to finish executing. Auto means that the agent startup script behavior configured in the workspace template is used.",
```

Comment:
It's probably worth mentioning the potential confusion in the description?

```suggestion
			Description: "Specifies whether or not to wait for the startup script to finish executing. Auto means that the agent startup script behavior configured in the workspace template is used. Not to be confused with CODER_SSH_WAIT.",
```

Reply:
I see why you would suggest this. I don't think we should do it, though. My reason for adding the comment about confusion is that right now config-ssh has both types of envs, but I would prefer all to be named `CONFIGSSH_`, so it's easier to reason about it.

I didn't make that change though because we don't have a way to deprecate or support multiple envs in clibase. It could be added manually but that should be a separate PR.